.github/workflows/url_checker.yml

```diff
@@ -0,0 +1,32 @@
+name: update url_allowlist.csv
+
+on:
+  schedule:
+    - cron: '0 0 1 * *' # At 00:00 on day-of-month 1
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout repo content
+        uses: actions/checkout@v2 # checkout the repository content
+
+      - name: setup python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10' # install the python version needed
+
+      - name: install python packages
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r test/requirements.txt
+
+      - name: execute py script # run url_checker.py
+        env:
+          DC_API_KEY: ${{ secrets.DC_API_KEY }}
+        run: python test/url_checker.py --update_allowlist
```

Comment:
Hmmm, do we want to update allowlist or do a test and fail if new things are added?

Reply:
I was thinking that for now this would just update the full list periodically (to catch both manifest url changes (which are checked via api) and urls that go stale, even if the files aren't manually modified)

I need to think of how to best test new urls (due to the syncing, since schema is usually added to g3 first). One option is to add another github action to run the test on PR submission and potentially block submission, but this could cause syncing issues, since the CL is submitted before the PR

Reply:
SG